bodhi/server/models.py

```diff
@@ -951,6 +957,37 @@ def from_tags(cls, tags, session):
             if release:
                 return release
 
+    def get_critpath_pkgs(self):
+        """
+        Get a list of package names that are critcal path for this Release.
+
+        Returns:
+            packages (list): A list of packages that are critical path for this Release.
+        """
+        return self.critpath_pkgs
```

Comment:
We don't really need a getter for this - it's conventional in Python to just directly use the attribute rather than a getter.